emcmake.py

```diff
@@ -15,7 +15,7 @@
 # Main run() function
 #
 def run():
-  if len(sys.argv) < 2 or sys.argv[1] in ('--version', '--help'):
+  if len(sys.argv) < 2 or sys.argv[1] in ('--version', '--help') or sys.argv[1] != 'cmake':
```

Comment:
I'm not sure this is good idea.  What about somebody who runs `emcmake /path/to/cmake` or `emcmake cmake.exe`

Reply:
Oh yea, good catch, removed.